apps/fluent-tester/src/E2E/MenuButtonExperimental/specs/ExperimentalMenuButton.spec.win.ts

```diff
@@ -0,0 +1,54 @@
+import NavigateAppPage from '../../common/NavigateAppPage.win';
+import ExperimentalMenuButtonPageObject from '../pages/ExperimentalMenuButtonPageObject.win';
+import { PAGE_TIMEOUT, BOOT_APP_TIMEOUT, MENUBUTTON_A11Y_ROLE } from '../../common/consts';
+import {
+  EXPERIMENTAL_MENU_BUTTON_ACCESSIBILITY_LABEL,
+  //EXPERIMENTAL_MENU_BUTTON_TEST_COMPONENT_LABEL,
+} from '../../../FluentTester/TestComponents/MenuButtonExperimental/consts';
+import { ComponentSelector } from '../../common/BasePage.win';
+
+// Before testing begins, allow up to 60 seconds for app to open
+describe('Experimental MenuButton Testing Initialization', function () {
+  it('Wait for app load', () => {
+    NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
+    expect(NavigateAppPage.isPageLoaded()).toBeTruthy();
+  });
+
+  it('Click and navigate to Experimental MenuButton test page', () => {
+    /* Scroll to component test page button in scrollview if not already visible*/
+    ExperimentalMenuButtonPageObject.scrollToComponentButton();
+    ExperimentalMenuButtonPageObject.waitForButtonDisplayed(PAGE_TIMEOUT);
+
+    /* Click on component button to navigate to test page */
+    NavigateAppPage.clickAndGoToExperimentalMenuButtonPage();
+    ExperimentalMenuButtonPageObject.waitForPageDisplayed(PAGE_TIMEOUT);
+
+    expect(ExperimentalMenuButtonPageObject.isPageLoaded()).toBeTruthy();
+  });
+});
+
+/* This will be re-enabled with a MenuButton Bug is fixed. Currently in PR - "Integrating accessibilityLabel functionality for MenuButton #1117" */
+describe('Experimental MenuButton Accessibility Testing', () => {
+  it('Experimental MenuButton - Validate accessibilityRole is correct', () => {
+    ExperimentalMenuButtonPageObject.scrollToTestElement();
+    ExperimentalMenuButtonPageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
+    expect(ExperimentalMenuButtonPageObject.getAccessibilityRole()).toEqual(MENUBUTTON_A11Y_ROLE);
+  });
+
+  it('Experimental MenuButton - Set accessibilityLabel', () => {
+    ExperimentalMenuButtonPageObject.scrollToTestElement();
+    ExperimentalMenuButtonPageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
+    expect(ExperimentalMenuButtonPageObject.getAccessibilityLabel(ComponentSelector.Primary)).toEqual(
+      EXPERIMENTAL_MENU_BUTTON_ACCESSIBILITY_LABEL,
+    );
+  });
+
+  // This test currently fails due to a bug (ADO #5681166). I will re-enable this test once it gets fixed
+  // it('Do not set accessibilityLabel -> Default to Experimental MenuButton label', () => {
```

Comment:
PR to fix is in now, so this should work. I verified that the text of the menu button was being read when I tested the change.

Reply:
Sounds good, thanks! I'll sync forward and add the test back